README.md

```diff
@@ -70,6 +87,14 @@ env.close()
 You can find [tutorials](https://github.com/myohub/myosuite/tree/main/docs/source/tutorials#tutorials) and the **ICRA2023 Colab Tutorial** [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1KGqZgSYgKXF-vaYC33GR9llDsIW9Rp-q)
 on how to load MyoSuite models/tasks, train them, and visualize their outcome. Also, you can find [baselines](https://github.com/myohub/myosuite/tree/main/myosuite/agents) to test some pre-trained policies.
 
+<!-- TODO : include ICRA 2024 tutorial? -->
```

Comment:
@raku-slyu - please note a pending ToDo here.

Reply:
ICRA-24 tutorial is updated